client/js/options.js

```diff
@@ -44,7 +44,7 @@ userOptions = null;
 module.exports = options;
 
 module.exports.shouldOpenMessagePreview = function(type) {
-	return (options.links && type === "link") || (options.thumbnails && type === "image");
+	return type === "link" ? options.links : options.media;
```

Comment:
Oh that's much cleaner. The former was so complicated I wasn't sure this did what it was meant to do :-P